gno.land/pkg/integration/testdata/use_work.txtar

```diff
@@ -0,0 +1,27 @@
+use gno.land/r/foobar/bar $WORK/bar 
+
+## start a new node
+gnoland start
+
+## execute Render
+gnokey maketx run -gas-fee 1000000ugnot -gas-wanted 2000000 -broadcast -chainid=tendermint_test test1 $WORK/script/script.gno
+
+## compare render
+stdout 'main: --- hello from foo ---'
+stdout 'OK!'
+stdout 'GAS WANTED: 200000'
+stdout 'GAS USED: '
+
+-- bar/bar.gno --
+package bar
+
+func Render(path string) string {
+ 	return "hello from foo"
+}
+
+-- script/script.gno --
+package main
+import "gno.land/r/foobar/bar"
+func main() {
+	println("main: ---", bar.Render(""), "---")
+}
```

Comment:
because other `.gno` files in txtar are formatted as well?
```suggestion
-- script/script.gno --
package main

import "gno.land/r/foobar/bar"

func main() {
	println("main: ---", bar.Render(""), "---")
}

```